Code-4-Community/speak-for-the-trees-frontend
# Site geo json query update

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/925293934/views/18904377/pulses/1398350639)



No longer uses the scientific species name for a tree, instead uses the common name. Also removes the fields that have been removed from the map site query in this PR https://github.com/Code-4-Community/speak-for-the-trees-backend-v2/pull/61

## This PR



Literally just removed fields and changed 1 name in 1 object and changed the property fetched for the tree popup


## Screenshots

 


![Capture](https://user-images.githubusercontent.com/19194912/122479926-e2fb5480-cf99-11eb-82a2-573a827720f6.PNG)

Notice all the new blue sites

## Verification Steps

 

I checked some trees popups to make sure their names were correct. Also for verification of the backend PR, I can see all of the new blue sites from inserting data earlier
